lib/rails_stdout_logging/rails.rb

```diff
@@ -1,5 +1,5 @@
 module RailsStdoutLogging
-  class StdoutLogger < ::Logger
+  class StdoutLogger < ::ActiveSupport::Logger
```

Comment:
The logger silence module is not defined there either, so I think it'd be OK to feature detect it (like it's done with the tagged logging below):

```ruby
class StdoutLogger < defined?(::ActiveSupport::Logger) ? ::ActiveSupport::Logger : ::Logger
```

@schneems any thoughts about that?